tide/master/localstreamer/CommandLineOptions.cpp

```diff
@@ -203,7 +193,7 @@ void CommandLineOptions::_parseCommandLineArguments( int& argc, char** argv )
     }
 
     _getHelp = vm.count( "help" );
-    _streamname = vm["streamname"].as<std::string>().c_str();
+    _streamId = vm["streamname"].as<std::string>().c_str();
```

Comment:
Yes, I will adapt further - the idea is to adjust to previous changes in deflect::Stream API which now uses id instead of name.